controllers/healthcheck.go

```diff
@@ -15,11 +15,12 @@ func Ping(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	_, err := w.Write([]byte("pong"))
 	if err != nil {
-		log.Err(err).Msg("Could not respond to Ping")
+		utils.LogErr(r, err, "Could not respond to Ping")
```

Comment:
Please add a `return` statement here to stop the controller after logging the error.